custom/icds_reports/templates/icds_reports/mobile_login.html

```diff
@@ -1,3 +1,12 @@
 {% extends "icds/login.html" %}
 {% load i18n %}
 {% block title %}{% trans "Login to ICDS CAS Mobile Dashboard" %}{% endblock title %}
+{% block page_content %}
+{{ block.super }}
+<script type="text/javascript">
+  document.addEventListener("DOMContentLoaded", function () {
+    var redirectUrl = "{{ redirectUrl }}";
+    window.history.replaceState(null, "", "?next=" + redirectUrl);
```

Comment:
This does a redirect? I thought this just replaced the history so that when you go to the next page and then press back, you'll go to the updated page. But I'm not that familiar with the history methods.

Reply:
Sorry this doesn't do a redirect, it just changes the URL (without touching history so back button isn't affected). But then that URL is used on login form submission to actually route properly on successful authentication.

Reply:
got it, thanks for explaining